app/controllers/users_controller.rb

```diff
@@ -52,6 +52,8 @@ def destroy
     def set_user
       @user = User.find_by id: params[:id]
       return redirect_to users_path if @user.nil?
+      @contents = @user.microposts.map { |micropost| micropost.content }
+      @contents = @contents.length > 0 ? @contents.join(', ') : "No content"
```

Comment:
Em viết 1 hàm nữa, rồi gọi như sau
```suggestion
def set user
      ...
      @contents = get_contents
end
private
def get_contents
      contents = @user.microposts.map { |micropost| micropost.content }.join(', ')
      # Chỗ này nhớ định nghĩa i18n k lại thêm 1 cmt nữa
      return i("no_content") if content.emply?
      
      content
end
```